### DISCUSSION TITLE
Computing project misc

### DISCUSSION THREAD

**Discussion by Takodachi6969** [Part-II-Computational-Physics/.github#103] [General]
After I have done my computing project, what admin do I need to do on my github page? Do I need to write a tutorial on how to use the notebook, which files to open? and do I need to change the name of the repository?

**Comment by Takodachi6969** [comment]
What admin do I need to do on the github page.... or am I fine leaving it with all the files

**Reply by AustenLamacraft** [reply]
I've given you admin rights and you should now be able to change the name of the repo (though it's not necessary). As to the structure of the repo, you may want to explicitly say in the README which file is the report, though in this case it's reasonably. 

Personally I'd tidy a little bit into a "data" folder but it's up to you
